setup.py

```diff
@@ -1,20 +1,17 @@
 from setuptools import setup
 
-readme = open('README.rst').read()
 history = open('CHANGES.rst').read().replace('.. :changelog:', '')
 
 setup(
-    name='sqlalchemy-redshift',
+    name='gx-sqlalchemy-redshift',
     version='0.8.15.dev0',
-    description='Amazon Redshift Dialect for sqlalchemy',
-    long_description=readme + '\n\n' + history,
+    description='Forked Amazon Redshift Dialect for sqlalchemy',
```

Comment:
It would be good to add our name to the description. Something like:
Great Expectations fork of the Amazon Redshift sqlalchemy dialect